scripts/variantstore/wdl/GvsValidateVAT.wdl

```diff
@@ -148,61 +210,67 @@ task SpotCheckForExpectedTranscripts {
         cpu: "1"
         disks: "local-disk 100 HDD"
     }
-    # ------------------------------------------------
-    # Output: {"Name of validation rule": "PASS/FAIL plus additional validation results"}
+
     output {
         Map[String, String] result = {"SpotCheckForExpectedTranscripts": read_string('validation_results.txt')}
     }
 }
 
-task GetBQTableLastModifiedDatetime {
-    # because this is being used to determine if the data has changed, never use call cache
-    meta {
-        volatile: true
-    }
-
+task SchemaOnlyOneRowPerNullTranscript {
     input {
-        String query_project
-        String fq_table
+        String query_project_id
+        String fq_vat_table
         String? service_account_json_path
+        String last_modified_timestamp
     }
 
     String has_service_account_file = if (defined(service_account_json_path)) then 'true' else 'false'
 
-    # ------------------------------------------------
-    # try to get the last modified date for the table in question; fail if something comes back from BigQuery
-    # that isn't in the right format (e.g. an error)
     command <<<
         set -e
 
         if [ ~{has_service_account_file} = 'true' ]; then
             gsutil cp ~{service_account_json_path} local.service_account.json
             gcloud auth activate-service-account --key-file=local.service_account.json
-            gcloud config set project ~{query_project}
+            gcloud config set project ~{query_project_id}
         fi
+        echo "project_id = ~{query_project_id}" > ~/.bigqueryrc
 
-        echo "project_id = ~{query_project}" > ~/.bigqueryrc
+        bq query --nouse_legacy_sql --project_id=~{query_project_id} --format=csv 'SELECT * FROM (SELECT
```

Comment:
I think this could us a HAVING instead of the nested query… like

```
SELECT vid
FROM 
    ~{fq_vat_table}
WHERE
    transcript_source is NULL AND
    transcript is NULL
GROUP BY vid
HAVING count(vid) > 1
```